### PR TITLE
20 - Adding routes

### PR DESCRIPTION
**Pull request by aquintiliano**
…eated

### CODE REVIEW

**Comment by ghost** [comment]
Waiting for review from @lauroaraujo 

Also, please fix conflicts.

**Comment by aquintiliano** [comment]
conflicts fixed

File: Frontend/src/app/app.component.html [hunk 1]
```diff
@@ -1,8 +1,3 @@
-<div>
-	<div>
-		<app-login (onLoginSuccess)="loginSuccess($event)"></app-login>
-	</div>
-	<div>
-		<app-create-player (onCreatePlayerSuccess)="createPlayerSuccess($event)"></app-create-player>
-	</div>
-</div>
+<h1>
```
Comment (lauroaraujo) [inline Frontend/src/app/app.component.html]: Why is the router outlet inside a header (h1)? Seems odd...

File: Frontend/src/app/page-not-found/page-not-found.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<p>
```
Comment (lauroaraujo) [inline Frontend/src/app/page-not-found/page-not-found.component.html:1]: The h1 tag from the app component should probably be here

**Review by lauroaraujo — CHANGES_REQUESTED** [review]
Looks fine, only one minor comment

**Comment by aquintiliano** [comment]
merging with Lauro authorization

**Pull request merged**